### PR TITLE
Style guide

### PR DESCRIPTION
**Pull request by kfrn**
Addresses #429 

* Adds guidelines for contributing to readme (mostly nicked from @kieranjol :)
* Adds a PR template

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+## Checklist
+
+* [ ] I've referred to the the [Guidelines for contributing](https://github.com/amiaopensource/ffmprovisr/blob/gh-pages/readme.md#guidelines-for-contributing)
+* [ ] New recipe has been added to `recipes.txt`
```
Comment (retokromer) [inline .github/pull_request_template.md]: I am not sure about the second point. Would this not confuse a person making her/his first contribution?

I am happy to sync form time to time, as I did in the past. (There is a on-liner here, but I personally use a short Perl script to gather the data from GitHub, to generate the correct output for FFCommand_Engine and to install it directly at the right place.)

Comment (kieranjol) [reply .github/pull_request_template.md]: Thanks for clarifying Reto - I'm happy to remove it. I wasn't really familiar with the recipes.txt protocol and saw a recent commit you made and so I added it in.

**Comment by kieranjol** [comment]
Hi @kfrn thanks so much - i love the structure here. Other folks might have other suggestions but I'm happy with it, and I take Reto's point about removing the recipes.txt point..

File: .github/pull_request_template.md (hunk 1 above)
Comment (retokromer) [reply .github/pull_request_template.md]: I could add here a new document explaining what is FFCommand_Engine, how to compile its last version and how to import ffmprovisr’s recipe into it.

**Comment by privatezero** [comment]
This looks good to me as well!

**Comment by kfrn** [comment]
Cheers! I'll remove the part about `recipes.txt` when I'm home this evening :)

**Comment by kfrn** [comment]
Now updated :)

**Review by retokromer — APPROVED** [review]

**Review by kieranjol — APPROVED** [review]
This is so wonderful, I LOVE the checklist! Next level! Thank you!

**Review by ablwr — APPROVED** [review]

**Comment by kfrn** [comment]
Thanks everyone! :)

**Pull request merged**